.travis.yml

```diff
@@ -150,38 +136,32 @@ install:
 
     # GLFW: version 2 shouldn't work (so let's try on Py2.6), version 3 will
     # We can't use most recent builds because our CMake is too old :(
-    - if [ "${PYTHON}" == "2.6" ] && [ "${DEPS}" == "full" ]; then
-        travis_retry sudo apt-get -qq install libglfw2 > ${REDIRECT_TO};
-      fi
     - if [ "${PYTHON}" != "2.6" ] && [ "${DEPS}" == "full" ]; then
-        travis_retry sudo apt-get -qq install xorg-dev libglu1-mesa-dev > ${REDIRECT_TO};
         git clone git://github.com/glfw/glfw.git &> ${REDIRECT_TO};
         cd glfw;
         git checkout 5b6e671;
-        cmake -DBUILD_SHARED_LIBS=true -DGLFW_BUILD_EXAMPLES=false -DGLFW_BUILD_TESTS=false -DGLFW_BUILD_DOCS=false . > ${REDIRECT_TO};
-        sudo make install > ${REDIRECT_TO};
+        cmake -DCMAKE_INSTALL_PREFIX=$HOME -DBUILD_SHARED_LIBS=true -DGLFW_BUILD_EXAMPLES=false -DGLFW_BUILD_TESTS=false -DGLFW_BUILD_DOCS=false . &> ${REDIRECT_TO};
+        make install &> ${REDIRECT_TO};
+        cd ~;
+        export GLFW_LIBRARY=${HOME}/lib/libglfw.so;
       fi
 
     # Install SDL2 (on 2.6, let's only install sdl2-python to test handling)
     - if [ "${DEPS}" == "full" ]; then
         if [ "${PYTHON}" != "2.6" ]; then
-          cd ~;
-          travis_retry sudo apt-get -qq -y install mercurial libdbus-1-dev libgl1-mesa-dev libglu1-mesa-dev libpulse-dev libx11-dev libxcursor-dev libxext-dev libxi-dev libxinerama-dev libxrandr-dev libxss-dev libxt-dev libxv-dev libxxf86vm-dev libasound2-dev libts-dev libudev-dev > ${REDIRECT_TO};
           wget -q http://www.libsdl.org/release/SDL2-2.0.3.tar.gz;
           tar xzf SDL2-2.0.3.tar.gz &> ${REDIRECT_TO};
           cd SDL2-2.0.3;
           mkdir build;
           cd build;
-          ../configure --disable-audio --disable-render --disable-joystick --disable-haptic --disable-power --disable-cpuinfo &> ${REDIRECT_TO};
+          ../configure --prefix=$HOME --disable-audio --disable-render --disable-joystick --disable-haptic --disable-power --disable-cpuinfo &> ${REDIRECT_TO};
```

Comment:
I've tried setting LD_LIBRARY_PATH, that doesn't work. We used to use standard install directories and use `sudo ldconfig`, but we can't do that anymore.